assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -230,6 +292,11 @@ export default function ChipTabGroup( { allMetricItems, savedItemSlugs } ) {
 									key={ group.SLUG }
 									slug={ group.SLUG }
 									label={ group.LABEL }
+									hasNewBadge={
+										!! newlyDetectedMetrics?.[
+											group.SLUG
+										] ?? false
```

Comment:
You're already forcing a boolean here, so no need to use nullish coalescing.
```suggestion
										!! newlyDetectedMetrics?.[
											group.SLUG
										]
```